app/views/blogs/new.html.haml

```diff
@@ -0,0 +1,21 @@
+- if user_signed_in?
```

Comment:
Remove this validation since it is already authenticated in the blogs_controller before_filter. @jedilaunchlabs